spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET #index each user posts' do
+    before :each do
+      get '/users/1/posts'
+    end
+    it 'returns http success' do
+      expect(response).to have_http_status(:success)
+    end
+    it 'checks if the correct template was rendered' do
+      expect(response).to render_template(:index)
```

Comment:
* Kindly add tests for the show actions as mentioned in the [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/controller_specs_project.md#:~:text=Make%20sure%20that%20for%20each%20action%20you%20check)